src/seml/start.py

```diff
@@ -123,6 +126,13 @@ def get_command_from_exp(
                 f"Starting debug server with IP '{ip_address}' and port '{port}'. "
                 f'Experiment will wait for a debug client to attach.'
             )
+            attach_link = _generate_debug_attach_url(ip_address, port)
+
+            logging.info(
+                "If you are using VSCode, you can use the 'Debug Launcher' extension to attach: \n"
+            )
+            console.out(attach_link)
```

Comment:
shouldn't there be a line ending afterwards?

Reply:
console.out ends its lines with \n automatically